docker-compose.yml

```diff
@@ -145,3 +145,8 @@ services:
     depends_on:
       - ckan
       - giftless
+    hostname: adr
+    networks:
+      default:
+        aliases:
+          - adr
```

Comment:
Strangely, I was convinced that `hostname` field alone did the job of putting the nginx container at the http://adr/ domain on the docker network, however coming back to it this morning that didn't work.  Not sure how I thought it worked yesterday.  Upon reading further setting the container's alias on the network achieves the intended result.